scripts/.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file looks like it might have been mistakenly checked in?

Reply:
hmm . seems like it. deleted the file!